playbooks/app_install/roles/docker_install/tasks/main.yml

```diff
@@ -43,13 +43,14 @@
     state: directory
     mode: 750
 
-- name: Touch /etc/docker/daemon.json if it doesnt exist or update mod time
-  ansible.builtin.file:
+- name: Configure Docker to use custom data directory
+  lineinfile:
```

Comment:
when you have a chance, just add the collection path, will be better off in long run
ansible.builtin.lininfile